docs/version0.44.md

```diff
@@ -76,6 +77,12 @@ The `jcmd` tool also now supports specifying `0` as a VMID to target all VMs.
 
 For more information, see [Java diagnostic command (`jcmd`) tool](tool_jcmd.md).
 
+### Change in the Direct Dump Reader plug-ins code
```

Comment:
I suggest "the flag names that are used in the DDR code to refer to the fields of `J9BuildFlags` are changed" or similar be the title of the What's New. User's shouldn't have to read the big description to figure out what changed.